sublime-package.json

```diff
@@ -0,0 +1,417 @@
+{
+  "contributions": {
+    "settings": [
+      {
+        "schema": {
+          "$id": "sublime://settings/LSP",
+          "definitions": {
+            // User preferences that are shared with "Preferences.sublime-settings"
+            "lsp_format_on_save": {
+              "type": "boolean",
+              "default": false,
+              "markdownDescription": "Run the server's formatProvider (if supported) on a document before saving. This option is also supported in syntax-specific settings and/or in the `\"settings\"` section of project files."
+            },
+            "ClientInitializationOptions": {
+              "type": "object",
+              "markdownDescription": "The initializationOptions that are passed to the language server process during the _initialize_ phase. This is a rather free-form dictionary of key-value pairs and is different per language server. Look up documentation of your specific langauge server to see what the possible key-value pairs can be."
+            },
+            "ClientSettings": {
+              "type": "object",
+              "markdownDescription": "The server settings. This is a rather free-form dictionary of key-value pairs and is different per language server. Look up documentation of your specific langauge server to see what the possible key-value pairs can be."
+            },
+            "ClientExperimentalCapabilities": {
+              "type": "object",
+              "markdownDescription": "Experimental capabilities that are passed to the language server process during the _initialize_ phase. This is different per language server, so you'll need to look up documentation for your particular language server."
+            },
+            "ClientEnv": {
+              "type": "object",
+              "markdownDescription": "Specify environment variables to pass to the language server process on startup.",
+              "additionalProperties": {
+                "type": "string",
+                "markdownDescription": "The value for this environment variable."
+              },
+            },
+            "ClientLanguages": {
+              "markdownDescription": "An array which defines to which views this language server should be attached.",
+              "type": "array",
+              "minItems": 1,
+              "items": {
+                "$ref": "#/definitions/LanguageConfig"
+              },
+            },
+            "LanguageConfig": {
+              "type": "object",
+              "markdownDescription": "A language configuration defines which servers are attached to which views",
+              "properties": {
+                "languageId": {
+                  "type": "string",
+                  "default": "somelang",
+                  "markdownDescription": "The \"Language ID\". It is sent to the language server so that it knows with what kind of file it is dealing with."
+                },
+                "document_selector": {
```

Comment:
This needs to be removed.

Reply:
I left it on purpose since `LSP-*` packages that want to be compatible with both ST3 and ST4 need to have both variants. And not including it in the schema would show an error due to `additionalProperties: false`.

Reply:
Ah okay. I guess that makes sense. Note however that vscode-json will now suggest this property as a completion.

Reply:
It's marked with `**ST4 only**` comment BTW.

Reply:
Actually there is a VSCode extension `doNotSuggest` that I could use here.

Reply:
Fixed directly on master: https://github.com/sublimelsp/LSP/commit/fa750cfec4bfc0a89cbddd3011320bb72f6db51c